DashAI/back/main.py

```diff
@@ -1,42 +1,56 @@
-import json
+import logging
+import sys
 
 import uvicorn
-from fastapi import Body, FastAPI
-from fastapi.middleware.cors import CORSMiddleware
+from fastapi import FastAPI, Request
+from fastapi.responses import RedirectResponse
+from fastapi.staticfiles import StaticFiles
+from fastapi.templating import Jinja2Templates
+from sqlalchemy.exc import DBAPIError, SQLAlchemyError
+from sqlalchemy.sql import text
 
-from DashAI.back.config_object import ConfigObject
 from DashAI.back.database import db
-from DashAI.back.models.classes.getters import filter_by_parent
-from DashAI.back.models.enums.squema_types import SquemaTypes
 from DashAI.back.routers import datasets, experiments
-from DashAI.back.routers.session_class import session_info
 
 # TODO These imports should be removed because they are unused, but currently needed.
 from DashAI.back.tasks.tabular_classification_task import TabularClassificationTask
 from DashAI.back.tasks.task import Task
 from DashAI.back.tasks.text_classification_task import TextClassificationTask
 from DashAI.back.tasks.translation_task import TranslationTask
 
+logging.basicConfig(level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
 app = FastAPI(title="DashAI")
 
-origins = [
-    "http://localhost:3000",
-]
 
-app.add_middleware(
-    CORSMiddleware,
-    allow_origins=origins,
-    allow_credentials=True,
-    allow_methods=["*"],
-    allow_headers=["*"],
-)
+@app.get("/")
+async def redirect():
+    return RedirectResponse(url="/app")
+
+
+api = FastAPI(title="DashAI API")
+
+api.include_router(datasets.router)
+api.include_router(experiments.router)
+
+app.mount("/static", StaticFiles(directory="DashAI/front/build/static"), name="static")
+app.mount("/images", StaticFiles(directory="DashAI/front/build/images"), name="images")
+templates = Jinja2Templates(directory="DashAI/front/build")
```

Comment:
Why use jinja if there are no templates to compile?

Reply:
I was following [this](https://stackoverflow.com/a/64147628) answer on stackoverflow.
You are correct. There was an easier way
```
@app.get("/app/{full_path:path}")
async def read_index():
    return FileResponse('DashAI/front/build/index.html')
```

Reply:
We also now load statics without the app.mount black box and redirects if the file is not found.
```
@app.get("/{full_path:path}")
async def serve_files(full_path: str):
    try:
        if full_path == "":
            return RedirectResponse(url="/app/")
        path = f"DashAI/front/build/{full_path}"
        os.stat(path)  # This checks if the file exists
        return FileResponse(path)  # You can't catch the exception here
    except FileNotFoundError:
        return RedirectResponse(url="/app/")
```

Reply:
Nice!